docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -335,7 +335,7 @@ There are some known upstream Linux Kernel and firmware issues that can affect t
 
 #### Debian And Ubuntu Linux
 
-The `jellyfin-ffmpeg5` deb package required by Jellyfin 10.8 comes with all necessary user mode Intel media drivers.
+Using jellyfin-ffmpeg with Intel Quick Sync requires additional Intel media driver packages.
```

Comment:
I assume this change is no longer needed? Our `jellyfin-ffmpeg*` deb package does ensure that all Intel QSV packages except OpenCL are installed correctly. When 10.9 comes out we need to replace 10.8->10.9 and ffmpeg5 -> ffmpeg6.